snorkel/classification/snorkel_classifier.py

```diff
@@ -371,22 +384,36 @@ def score(self, dataloaders: List["DictDataLoader"]) -> Dict[str, float]:
         self.eval()
 
         metric_score_dict = dict()
+        # By default, evaluate all labels on corresponding task of same name
 
         for dataloader in dataloaders:
             results = self.predict(dataloader, return_preds=True)
-            for task_name in results["golds"].keys():
-                metric_scores = self.scorers[task_name].score(
-                    results["golds"][task_name],
-                    results["preds"][task_name],
-                    results["probs"][task_name],
+
+            # By default, evaluate each label_name on task with corresponding task_name
+            eval_dict = {name: name for name in results["golds"].keys()}
```

Comment:
would `.get(task_name, task_name)` below be cleaner than forming this dict? or would this produce a better error message for a missing name?

Reply:
unfortunately, `remap_labels` is not exhaustive; it only includes the necessary overrides for certain labels. 
we also need to iterate through labels as keys, so we'd like to first populate the generic labels, and then override with those from `remap_labels`

Reply:
ok sg